doc/source/contrib/language_ref/property_ref/requirement_types.rst

```diff
@@ -79,19 +79,20 @@ Cache keys:
 Snap
 ----
 
-Takes an apt package name and optional list of revision ranges. Returns True if
-the package exists and if provided, revision is within ranges.
```

Comment:
Thank you for pointing this out. I have gone ahead and fixed it in https://github.com/canonical/hotsos/commit/ef40c2ef8961239a493a8deeab79a0c5dd9715f7 since both the apt and snap property docs needed tweaking. You can go ahead and rebase your branch to get the fix (or just remove your change).